Packs/Anomali_ThreatStream/ReleaseNotes/2_2_5.md

```diff
@@ -0,0 +1,6 @@
+
+#### Playbooks
+
+##### Detonate URL - ThreatStream
+
+- Fixed an issue where the playbook output listed **'ANYRUN'** instead of **'ThreatStream'** command output.
```

Comment:
please also mention that you changed the Default value for the PB input - which now will take the URL.Data info instead of just the URL object from the context.